_im-plugin/ism/api.md

```diff
@@ -546,3 +546,154 @@ DELETE _plugins/_ism/policies/policy_1
   "_primary_term": 1
 }
 ```
+
+## Error prevention validation
+Introduced 2.4
+{: .label .label-purple }
+
+ISM allows you to execute an action automatically. However, action execution can fail for a variety of reasons. Error prevention validation checks the possible reason for a failure before execution of an action.
+
+To enable error prevention validation, set the `plugins.index_state_management.validation_service.enabled` setting to `true`:
+
+#### Request
```

Comment:
I would remove the heading "request" here and just include the request after the colon.